lib/branding.rb

```diff
@@ -0,0 +1,19 @@
+# Provides a helper utility for loading branding configs.
+module Branding
+
+  module_function
+
+  # Loads branding config from YAML file.
+  #
+  # @param keys [Array<Object>] A list of the keys to return configs for.
+  #
+  # @example Return a value
+  #   Branding.fetch(:settings, :should_work) # => true
+  #   Branding.fetch(:settings, :email) # => 'user@example.com'
+  #   Branding.fetch(:settings, :missing) # => nil
+  # @return [Object] The value of the config
+  def fetch(*keys)
```

Comment:
Thank you this is so much nicer than the old `Rails.configuration.branding['blah']['blah']['blah']` :)